### ISSUE TITLE
RISG -- Wrong Text Characters being Displayed

### ISSUE THREAD

**Issue by alandesbonnet** [ecsion/eSeaGrant-Client-Bug-Tracker#625]
at this URL

http://eseagrant.gso.uri.edu/index.php?module=RFPProposal&view=Detail&record=7628&mode=showMultiQuestionStem&QuestionId=6109&tab_label=Funding%20Request&tab_label=Funding%20Request

incorrect text characters are being displayed in each line as shown in the following screenshot. This must be corrected to remove confusion. Thank you.

![screen shot 2017-03-12 at 10 47 08 am](https://cloud.githubusercontent.com/assets/18398763/23832732/7df8fc72-0711-11e7-99f1-5a5d10e28cfb.png)

**Comment by ecsion** [comment]
Noticed that is showing proper text now. What did you change?

**Comment by alandesbonnet** [comment]
I went in and edited the text in the custom form. That said, there seems to be a regular inconsistency in the way eSeaGrant shows special characters, such as “ or ‘ or e (italicized text) and others. There appears to be some error occurring translating certain text characters across forms and screens. For instance,  see GitHub #626 where this seems to be occurring again, in all instances where I used e (italicized e) in the text. 

This needs to be corrected so that these characters can be used without these nasty side effects. Thank you.

Alan Desbonnet
Assistant Director
